src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -389,9 +392,14 @@ def _get_extension_suppressions(mod_loaders):
         self.command_group_table.clear()
         self.command_table.clear()
 
+        need_skip_command_index = False
+        if isinstance(args, list) and args:
+            need_skip_command_index = args[0] in SKIP_COMMAND_INDEX_FOR
```

Comment:
I have made `load_command_table` idempotent. You may load command table again with command index disabled to get the full command table.

```py
# pseudocode
os.environ['AZURE_CORE_USE_COMMAND_INDEX']='True'
load_command_table()
```

Reply:
OK~